src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineFunctions.scala

```diff
@@ -338,6 +372,14 @@ object PipelineFunctions {
       s"modules include ${pipelineModules.map(m => s"\n(${m.moduleId}, ${m.moduleName})").mkString("\n")}"))
   }
 
+  private[overwatch] def deriveSKU(isAutomated: Column, sparkVersion: Column): Column = {
+    val isJobsLight = sparkVersion.like("apache_spark_%")
+    when(isAutomated && isJobsLight, "jobsLight")
+      .when(isAutomated && !isJobsLight, "automated")
+      .when(!isAutomated, "interactive")
```

Comment:
what about Databricks SQL?

Reply:
Yep it was in there but no longer belongs here since the entire DBSQL has move over to endpoints now. So the logic is no longer necessary here.